doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -31,13 +31,19 @@ The following sections provide detailed lists of changes by component.
 nRF9160
 =======
 
+* Updated:
+
   * :ref:`lwm2m_client` sample:
 
     * Added support for Thingy:91.
     * Added more LwM2M objects.
     * LwM2M sensor objects now uses the actual sensors available to the Thingy:91. If the nRF9160 DK is used, it uses simulated sensors instead.
     * Added possibility to poll sensors and notify the server if the measured changes are large enough.
 
+  * A-GPS library:
```

Comment:
```suggestion
  * :ref:`lib_nrf_cloud_agps` library:
```

Reply:
A-GPS library is different from nRF Cloud library. There doesn't seem to be documentation for the A-GPS library (API in include/modem/agps.h), that's the reason I didn't use a link.

Reply:
Oh, sorry for that then. Do you think some documentation should be added for that library then?

Reply:
I guess it would be good to have documentation for it. Fortunately, it's a simple library so the documentation shouldn't be too much work.